frontend/public/components/storage/create-pvc.tsx

```diff
@@ -163,23 +180,29 @@ export class CreatePVCForm extends React.Component<CreatePVCFormProps, CreatePVC
         </label>
         <div className="form-group">
           {this.state.accessModeRadios.map(radio => {
-            const disabled = !!storageClassAccessMode;
-            const checked = !disabled ? radio.value === accessMode : radio.value === storageClassAccessMode;
-            return (
-              <RadioInput
-                {...radio}
-                key={radio.value}
-                onChange={this.handleChange}
-                inline={true}
-                disabled={disabled}
-                checked={checked}
-                aria-describedby="access-mode-help"
-                name="accessMode"
-              />
-            );
+            let radioObj = null;
+            const disabled = storageClassAccessModes.indexOf(radio.value) !== -1 ? false : true;
+
+            storageClassAccessModes.forEach(mode => {
```

Comment:
I'm not sure this logic is correct. Won't it always set the checked value of the radio to the last access mode in `storageClassAccessModes`?

Reply:
It will work now, since access mode has been set to default with the change of storage class selection